samples/bluetooth/iso_connected_benchmark/src/main.c

```diff
@@ -58,8 +58,9 @@ static size_t total_iso_conn_count;
 static uint32_t iso_send_count;
 static struct bt_iso_cig *cig;
 
-NET_BUF_POOL_FIXED_DEFINE(tx_pool, 1, CONFIG_BT_ISO_TX_MTU, NULL);
-static uint8_t iso_data[CONFIG_BT_ISO_TX_MTU - BT_ISO_CHAN_SEND_RESERVE];
+NET_BUF_POOL_FIXED_DEFINE(tx_pool, 1, BT_ISO_SDU_BUF_SIZE(CONFIG_BT_ISO_TX_MTU),
+			  NULL);
+static uint8_t iso_data[DEFAULT_CIS_SDU_SIZE];
```

Comment:
Same - data is DEFAULT_CIS_SDU_SIZE, but the buffer is sized for CONFIG_BT_ISO_TX_MTU

Reply:
See https://github.com/zephyrproject-rtos/zephyr/pull/38610#discussion_r710834141

Reply:
Same proposal - use `DEFAULT_CIS_PDU_SIZE` as the input to `BT_ISO_SDU_BUF_SIZE()` to make the connection clear.

Reply:
+1 on using only one definition.

Reply:
See https://github.com/zephyrproject-rtos/zephyr/pull/38610/files#r716025403